.github/workflows/update-geoip.yml

```diff
@@ -24,14 +16,25 @@ concurrency:
 jobs:
   certificate-check:
     name: "Check for updated GeoIP DB"
+
     # Don't run the cron job on forks.
-    if: ${{ github.event_name != 'schedule' || github.repository == 'YOURLS/YOURLS' }}
+    # Don't run on a branch that was created by the workflow
+    if: |
+      ${{ github.event_name == 'schedule' && github.repository != 'YOURLS/YOURLS' }} &&
+      ${{ github.ref != 'refs/heads/auto-update-geoip' }}
```

Comment:
```suggestion
```

All this block can now be dropped with confidence

Reply:
It will run, but it won't interact with upstream repo, and will actually fail because the secrets won't be available there.

Reply:
Yep, got that while working on ozh/YOURLS to lessen the noise here ;-)